generator/address_parser/tiger_parser.cpp

```diff
@@ -100,18 +100,3 @@ bool ParseLine(std::string_view line, AddressEntry & e)
 }
 
 } // namespace tiger
-
-namespace feature
-{
-std::string DebugPrint(InterpolType type)
-{
-  switch (type)
-  {
-  case InterpolType::None: return "Interpol::None";
-  case InterpolType::Any: return "Interpol::Any";
```

Comment:
What is printed here?

Reply:
```
"None"
"Any"
```

Reply:
It would be great to print either a short form or full form with enum name when necessary for clarity. Or rename enum values accordingly.